sdk/src/main/java/com/karhoo/sdk/api/network/client/APITemplate.kt

```diff
@@ -98,8 +99,12 @@ interface   APITemplate {
 
         const val PAYMENT_PROVIDERS_METHOD = "/v3/payments/providers"
         const val ADYEN_PUBLIC_KEY_METHOD = "/v3/payments/adyen/public-key"
```

Comment:
I think we can mark the public-key endpoint/method as deprecated since we won't be using it anymore (instead use the client key)